src/components/Footer.tsx

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import { MaxWidthWrapper } from "./Wrappers";
+import tw, { styled } from "twin.macro";
+import Link from "./Link";
+import Heading from "./Heading";
+import { ReactComponent as UnstyledLogo } from "../assets/logo.svg";
+import { ArrowLeft } from "../assets/icons";
+const footerLinks = [
+  {
+    name: "How UMA Works",
+    href: "https://docs.umaproject.org/getting-started/overview",
+  },
+  { name: "Docs", href: "https://docs.umaproject.org/" },
+  { name: "FAQS", href: "https://umaproject.org/faq.html" },
+  { name: "Contact", href: "mailto:hello@umaproject.org" },
+  {
+    name: "Getting Started",
+    href: "https://docs.umaproject.org/build-walkthrough/build-process",
+  },
+  { name: "Vote", href: "https://vote.umaproject.org/" },
+  { name: "Careers", href: "https://angel.co/risk-labs/jobs" },
+];
+
+const SubscribeForm: React.FC = () => {
+  return (
+    <Form>
+      <Input type="email" name="email" placeholder="Email Address" />
+      <button type="submit">
+        <ArrowLeft />
+      </button>
+    </Form>
+  );
+};
+
+const Footer: React.FC = () => {
+  return (
+    <MaxWidthWrapper as="footer">
+      <FooterWrapper>
+        <div>
+          <Link href="/">
+            <Logo />
+          </Link>
+          <FooterParagraph>Risk Labs © 2020</FooterParagraph>
+        </div>
+        <div>
+          <LinkList>
+            {footerLinks.map((link) => (
+              <ListItem key={link.name}>
+                <Link
+                  href={link.href}
+                  target="_blank"
+                  rel="noopener norefferrer"
+                >
+                  {link.name}
+                </Link>
+              </ListItem>
+            ))}
+          </LinkList>
+        </div>
+        <div>
+          <FooterHeading level={6}>Get UMA Updates</FooterHeading>
+          <FooterParagraph>
+            Sign up for our newsletter to stay updated about the UMA project.
+          </FooterParagraph>
+          <SubscribeForm />
+        </div>
+      </FooterWrapper>
+    </MaxWidthWrapper>
+  );
+};
+export default Footer;
+
+export const FooterWrapper = styled.div`
+  ${tw`w-full px-10 py-20 grid md:divide-x md:divide-gray`};
+  grid-template-columns: 24% 1fr 36%;
+  > div + div {
+    padding-left: 100px;
+  }
```

Comment:
could you give an explainer of how this works. I think theres a lot of specific knowledge you are using when definining styles.  You are using both styled components and tailwind to generate css + element, is that right? and what is `div + div`? also why is this exported, is it meant to be used generally?

Reply:
I'll try to break the answer down into three parts:

1. I am using emotion styled as the "styled components framework" and then integrating that with tailwind, this makes it possible to use tailwind classes inside styled components instead of having to write all the CSS manually.
2. The component here is exported because I needed to change how the `Link` component looks inside of it. So I export it here an import it back into the `Link` component to do this:
```javascript
// inside Link.tsx
import { FooterWrapper } from "./Footer"

const Wrapper = styled.a`
// Link styles...
  ${FooterWrapper} & {
    box-shadow: 0 0 0 #fff;
    ${tw`transition duration-100`}
    :hover {
      box-shadow: 0 2px 0 #fff;
    }
  }
`;
`
```

3. the `> div + div` is just a CSS selector and is used here to give all divs that are followed by another div inside the FooterWrapper a 100px padding-left